### DISCUSSION TITLE
便签页怎么打开是空白

### DISCUSSION THREAD

**Discussion by happyhrw** [MaLuns/hexo-theme-async#38] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/100116732/201003200-ff84c962-8ea6-47dc-82c9-a75ef989132f.png)

**Comment by MaLuns** [comment, marked as answer]
可以正常显示的  https://www.imalun.com/tags/  
检测配置对不对

**Reply by happyhrw** [reply]
麻烦问一下，具体配置在哪个地方？前端小白，我找不到在哪

**Reply by happyhrw** [reply]
https://github.com/happyhrw/async

**Reply by MaLuns** [reply]
版本不是最新的

**Reply by MaLuns** [reply]
> https://github.com/happyhrw/async

你更新到新的版本就可以了

**Reply by happyhrw** [reply]
作者，不能使用命令更新主题吗？npm install ....

**Reply by MaLuns** [reply]
> 作者，不能使用命令更新主题吗？npm install ....

可以用的，你需要用npm 安装
